app/views/users/_form.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= form_for @user do |form| %>
+  <% if @user.errors.any? %>
+    <h2><%= pluralize(@user.errors.count, 'error')%> prohibited this entry from being saved</h2>
+    <ul>
+      <% @user.errors.full_messages.each do |msg| %>
+        <li><%= msg %></li>
+      <% end %>
+    </ul>
+  <% end %>
+   <p>
+    <%= form.label :username %><br>
+    <%= form.text_field :username, placeholder: "Your login here" %>
+  </p>
+
+  <p>
+    <%= form.label :email %><br>
+    <%= form.email_field :email, placeholder: "example@example.com" %>
+  </p>
+
+  <p>
+    <%= form.label :password %><br>
+    <%= form.text_field :password, placeholder: "Your Password here" %>
+  </p>
```

Comment:
Make this a `password_field` rather than a `text_field`. It is the proper form field to use for passwords.